Deploy plumbing Tekton resources to robocat (#280)
tektoncd/plumbing: robocat/README.md

# Setting up the Robocat cluster from scratch

These are step by step instructions on how to setup the `robocat` cluster using
the available automation. The automation is based on Tekton, so it requires

> a cluster with Tekton deployed, which will run the tasks required to setup

Reviewer: ```suggestion
a "driver" cluster with Tekton deployed, which will run the tasks required to setup
```

Author: Done

> the `robocat` cluster. These "driver" cluster is the `dogfooding` cluster.

Reviewer: ```suggestion
the `robocat` cluster. This "driver" cluster is currently the `dogfooding` cluster.
```

Author: Done

cat <<EOF | kubectl --cluster gke_tekton-releases_us-central1-a_dogfooding create -f -
apiVersion: v1
kind: Secret
metadata:
  name: robocat-tektoncd-cadmin-token
type: Opaque
data:
  ca.crt: $CA_CRT
  token: $TOKEN
EOF
```

# Verify the robocat cluster resources

Obtain the URL of the cluster:

```
kubectl cluster-info
```

Ensure that the [`robocat-tekton-deployer`](https://github.com/tektoncd/plumbing/blob/5f9cb51b8530f9bfc5e97e235980767ae53cdec9/tekton/resources/cd/clusters.yaml#L57)
and the [`robocat-cadmin`](https://github.com/tektoncd/plumbing/blob/5f9cb51b8530f9bfc5e97e235980767ae53cdec9/tekton/resources/cd/clusters.yaml#L77) resources point to the correct URL of the cluster.
If not fix them in git and re-apply them to the `dogfooding` cluster.

# Wait... or not

Almost everything else is setup automatically via cronjobs scheduled in the
`dogfooding` cluster. Since the setup of the DNS entry and the creation of the
`ClusterIssuer` at the right time are still not automated, it's best for now
to run through the setup "manually" by triggering the various cronjobs
one by one, at least for the initial setup.
Future hanges to the resources will be deployed nightly from git.

> # Point your cluster config to `dogfooding`

Reviewer: My understanding is that this is part of running the CronJobs manually so second level heading  makes more sense 
```suggestion
## Point your cluster config to `dogfooding`
```

Author: Done

From this point on, most of the work will be done on the `dogfooding` cluster,
so switch your configuration to point to it:

```
kubectl config use-context gke_tekton-releases_us-central1-a_dogfooding
```

> # Run the cronjobs

Reviewer: ```suggestion
## Run the cronjobs
```

Author: Done
